osf/migrations/0193_migrate_registered_meta.py

```diff
@@ -45,7 +45,7 @@ def migrate_registration_registered_meta(state, schema):
     migrate_registrations(
         dry_run=False,
         rows='all',
-        RegistrationModel=state.get_model('osf', 'registration')
+        AbstractNodeModel=state.get_model('osf', 'abstractnode')
```

Comment:
Minor: Technically superfluous, as `state.get_model('osf', 'registration')` is already returning all types of AbstractNodes

Reply:
true true, but i don't want to confuse future devs who might reasonably think filtering on `type='osf.registration'` is superfluous